plugins/ca-kafka-local/README.md

```diff
@@ -0,0 +1,49 @@
+# CA Kafka Local Devbox Plugin
+
+At Culture Amp we use Kafka for sharing data between services, and the private [kafka-local](https://github.com/cultureamp/kafka-local) repo is how we run that locally.
+
+Use this plugin to interact with `kafka-local` in your service. You still need to launch Kafka on your own, but this plugin provides environment variables and tools to make it easy for your project to connect to `kafka-local`.
+
+What it provides:
+
+- Environment variables. See [plugin.json](./plugin.json) for which variables are supplied and their values. They match those used in `kafka-local`.
+- Process Compose job
+  - Will print information to the terminal about Kafka, once `kafka-local` is up and running.
+  - Allows you to add a `depends_on` clause so other services wait for `kafka-local`.
+
+## Usage
+
+To start Kafka
+
+- Use [Hotel CLI](https://github.com/cultureamp/hotel)
```

Comment:
Maybe link to https://cultureamp.atlassian.net/wiki/spaces/DE/pages/3342434338/Local+development+environments+LDEs or https://github.com/cultureamp/devbox-extras/tree/main?tab=readme-ov-file#tooling-install-for-local-development-environments is better than the Hotel repo? Since there is no install instructions in the README

Reply:
The [Hotel instructions](https://github.com/cultureamp/hotel?tab=readme-ov-file#installation) do link to the confluence page you had, I think that's enough for now.

I think over time most people looking at this will have Hotel installed and I think linking to the README where other docs stem from might be better.

I'm open to changing it later if you disagree, but I'm keen to merge now.